docs/install/docker.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: page
+title: "Apache Zeppelin Releases Docker Images"
+description: "This document contains instructions about making docker containers for Zeppelin. It mainly provides guidance into how to create, publish and run docker images for zeppelin releases."
+group: install
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+{% include JB/setup %}
+
+# Apache Zeppelin Releases Docker Images
+
+<div id="toc"></div>
+
+## Overview 
+This document contains instructions about making docker containers for Zeppelin. It mainly provides guidance into how to create, publish and run docker images for zeppelin releases.
+
+## Quick Start
+### Installing Docker
+You need to [install docker](https://docs.docker.com/engine/installation/) on your machine.
+
+### Creating and Publishing Zeppelin docker image 
+* In order to be able to create and/or publish an image, you need to set the **DockerHub** credentials `DOCKER_USERNAME, DOCKER_PASSWORD, DOCKER_EMAIL` variables as environment variables.
+ 
+* To create an image for some release use :
+`create_release.sh <release-version> <git-tag>`.
+* To publish the created image use :
+`publish_release.sh <release-version> <git-tag>`
+
+### Running a Zeppelin  docker image 
+
+* To start Zeppelin, you need to pull the zeppelin release image: 
```

Comment:
Could you add a newline below this line? Or it'll rendered like this 
![screen shot 2016-10-25 at 4 52 14 pm](https://cloud.githubusercontent.com/assets/10060731/19677264/6c0e61be-9ad3-11e6-83cf-e0e7126f4cf4.png)